### DISCUSSION TITLE
An error when using the theme

### DISCUSSION THREAD

**Discussion by ypwang0371** [alshedivat/al-folio#375] [Q&A]
Hello, I came across the is error when I used this theme for my website. I was wondering if anyone know how to solve this. Thank you!

The page build failed for the `master` branch with the following error:

The tag `twitter` on line 5 in `_posts/2020-09-28-twitter.md` is not a recognized Liquid tag. For more information, see https://docs.github.com/github/working-with-github-pages/troubleshooting-jekyll-build-errors-for-github-pages-sites#unknown-tag-error.

**Comment by ZeroAda** [comment]
Hi, I encounter the same problem. Do you solve it?

**Reply by ypwang0371** [reply]
Sorry I don't. I stop using this theme.

**Comment by alshedivat** [comment]
this error happens due to incorrect webpage deployment. please make sure to closely follow [deployment section in readme](https://github.com/alshedivat/al-folio#deployment).

**Reply by ZeroAda** [reply]
Hi, thanks. But I still cannot figure out what the problem is. I delete posts and deploy as the instructions.  Now the problem is:

![image](https://user-images.githubusercontent.com/59690636/158307858-1a369ec5-771c-4bac-95b7-16be8218abce.png)
